services/httpd/handler.go

```diff
@@ -1177,6 +1181,14 @@ func (h *Handler) logging(inner http.Handler, name string) http.Handler {
 		l := &responseLogger{w: w}
 		inner.ServeHTTP(l, r)
 		h.CLFLogger.Println(buildLogLine(l, r, start))
+
+		// Always log server errors.
```

Comment:
Can this comment be changed to "Log server errors"? There's no mechanism to log non-server errors so saying we always log server errors gives no contrast to any other type of logging we do for errors. Also, it seems to me like this should say, "Log server error if present in the HTTP header."